docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Jobs | Dagster
+description: A job is an executable graph of ops, with optional resources and configuration.
+---
+
+# Jobs
+
+Jobs are the main unit of execution and monitoring in Dagster. The core of a job is a [graph](/concepts/ops-jobs-graphs/graphs) of [ops](/concepts/ops-jobs-graphs/ops) connected via data dependencies.
+
+Ops are linked together by defining the dependencies between their inputs and outputs. An important difference between Dagster and other workflow systems is that, in Dagster, op dependencies are expressed as data dependencies, not just execution dependencies.
+
+This difference enables Dagster to support richer modeling of dependencies. Instead of merely ensuring that the order of execution is correct, dependencies in Dagster provide a variety of compile and run-time checks.
+
+Using jobs, you can:
+
+- Materialize a selection of [assets](/concepts/assets/software-defined-assets)
+- Determine op behavior using [resources](/concepts/resources) and [configuration](/concepts/configuration/config-schema)
+- Define [schedules](/concepts/partitions-schedules-sensors/schedules) to execute jobs at fixed intervals
+- Define [sensors](/concepts/partitions-schedules-sensors/sensors) to trigger jobs when external changes occur
+
+---
+
+## Relevant APIs
+
+| Name                                | Description                                                                                                                                                     |
+| ----------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject object="job" decorator /> | The decorator used to create a job.                                                                                                                             |
+| <PyObject object="JobDefinition" /> | A job definition. Jobs are the main unit of execution and monitoring in Dagster. Typically constructed using the <PyObject object="job" decorator /> decorator. |
+|                                     |                                                                                                                                                                 |
+
+---
+
+## Creating jobs
```

Comment:
Thinking this section through a little more deeply, I wonder if the right hierarchy here is:
* From software-defined assets
* Directly from ops
  * Using the `@job` decorator
  * From a graph